packages/fuels/tests/scripts.rs

```diff
@@ -97,6 +97,8 @@ async fn test_basic_script_with_tx_policies() -> Result<()> {
 }
 
 #[tokio::test]
+// Remove this test once the new encoding lands as the max_input will be irrelevant
```

Comment:
Make it a `TODO` and/or open an issue for this so we don't forget?

Reply:
added todo and issue: https://github.com/FuelLabs/fuels-rs/issues/1317